entrypoints/email-handler-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+echo "Starting the email client app"
+flask db upgrade
```

Comment:
If the db upgrade is already run in web app, I think we can remove it here to avoid potential race condition